lib/index.js

```diff
@@ -92,6 +92,29 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.patch(
+    `${accountPath}/status`,
+    ensureAuthenticated,
+    validate('account.setStatus'),
+    asyncHandler(async (req, res) => {
+      /**
+       * @func setStatus
+       * @param {Request} req
+       * @param {Response} res
+       * @description - takes an account id from the route
+       * and a status in the req.body and
+       * changes that status in the account's meta
+       * possible statuses are active and deleted
+       * @route /:account/status
+       */
+      const {account: id} = req.params;
+      const {actor} = req.user || {};
+      const {status} = req.body;
+      await brAccount.setStatus({actor, status, id});
+      res.status(200).send();
```

Comment:
I mentioned something along these lines earlier, but patch usually returns something other than 200 right?

Reply:
I do remember this being mentioned, however do not remember the code. I just checked and basically 200 or 204 seems to be what most people use. tell me and I'll update this and the other branch too.

Reply:
looks like 204 updating.

Reply:
agreed, if there is no body returned, 204 is appropriate.

Reply:
changed on both branches sorry about that.